scanpipe/tests/data/dependencies/resolved_dependencies_poetry_inspect_packages.json

```diff
@@ -1158,38 +1161,15 @@
           "source_packages": [],
           "bug_tracking_url": null,
           "primary_language": "Python",
-          "license_detections": [
-            {
-              "matches": [
-                {
-                  "score": 100.0,
-                  "matcher": "1-hash",
-                  "end_line": 1,
-                  "rule_url": "https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/rules/spdx_license_id_apache-2.0_for_apache-2.0.RULE",
-                  "from_file": null,
-                  "start_line": 1,
-                  "matched_text": "apache-2.0",
-                  "match_coverage": 100.0,
-                  "matched_length": 3,
-                  "rule_relevance": 100,
-                  "rule_identifier": "spdx_license_id_apache-2.0_for_apache-2.0.RULE",
-                  "license_expression": "apache-2.0",
-                  "spdx_license_expression": "Apache-2.0"
-                }
-              ],
-              "identifier": "apache_2_0-d66ab77d-a5cc-7104-e702-dc7df61fe9e8",
-              "license_expression": "apache-2.0",
-              "license_expression_spdx": "Apache-2.0"
-            }
-          ],
+          "license_detections": [],
```

Comment:
I think these (and other license detection related changes) are not correct to remove, this is also failing in the tests for the same reason.

Reply:
Thanks, this is a side effect of the data/fixtures regen. 
Would you mind to regen the test data on your side?

Reply:
@tdruez Done!